src/psyclone/tests/psyGen_test.py

```diff
@@ -4158,7 +4158,44 @@ def test_fparser2astprocessor_handling_Case_construct(f2008_parser):
     assert len(ifnode.else_body[0].children) == 2  # SELECT CASE ends here
 
 
-def test_fparser2astprocessor_handling_invalid_Case_construct(f2008_parser):
+def test_fp2astproc_case_default(f2008_parser):
+    ''' Check that the fparser2ASTProcessor handles SELECT blocks with
+    a default clause. '''
+    from fparser.common.readfortran import FortranStringReader
+    from fparser.two.Fortran2003 import Execution_Part, Assignment_Stmt
+    case_clauses = ["CASE default\nbranch3 = 1\nbranch3 = branch3 * 2\n",
+                    "CASE (label1)\nbranch1 = 1\n",
+                    "CASE (label2)\nbranch2 = 1\n"]
+    # Loop over the 3 possible locations for the 'default' clause
+    for idx1, idx2, idx3 in [(0,1,2), (1,0,2), (1,2,0)]:
```

Comment:
Nice test :)

Reply:
But pylint complains about the missing whitespace separation after comas.

Reply:
Sigh :-)